packages/grafana-schema/src/raw/dashboard/x/dashboard_types.gen.ts

```diff
@@ -349,7 +349,7 @@ export type DashboardLinkType = ('link' | 'dashboards');
  * `custom`: Define the variable options manually using a comma-separated list.
  * `system`: Variables defined by Grafana. See: https://grafana.com/docs/grafana/latest/dashboards/variables/add-template-variables/#global-variables
  */
-export type VariableType = ('query' | 'adhoc' | 'groupby' | 'constant' | 'datasource' | 'interval' | 'textbox' | 'custom' | 'system');
+export type VariableType = ('query' | 'adhoc' | 'groupby' | 'constant' | 'datasource' | 'interval' | 'textbox' | 'custom' | 'system' | 'snapshot');
```

Comment:
Do we have any way to skip this variable from here? 

The goal for the schema is to represent a shareable save model, and the `snapshot` variable is just a runtime variable used to represent read-only variable data. Same for the `system` var.

Do we have any way to skip it from here?

Reply:
🤔 I didn't find a way to skip it from here